### DISCUSSION TITLE
Switching to Gas after setup with Electric complete, auto update?

### DISCUSSION THREAD

**Discussion by hotnuts21** [BottlecapDave/HomeAssistant-OctopusEnergy#149] [Q&A] [answered]
HI, 
We have been using Octopus Electric Agile for a while and this addon for a few weeks. We have just this week switched our Gas to Octopus (we were on a fix before).  The switch is complete but no gas sensors are showing. Should the integration pick this up automatically or do we need to do something to start pulling the Gas sensor data?

**Comment by BottlecapDave** [comment, marked as answer]
A restart of either Home Assistant or the account in the integrations section should cause the new gas sensors to appear (assuming the API is up to date)
